home-manager/ssh.nix

```diff
@@ -11,6 +11,23 @@ let
   # - id_*.pub: I CAN register them for different services.
 in
 {
+  # https://github.com/nix-community/home-manager/blob/master/modules/services/ssh-agent.nix
+  services.ssh-agent.enable = if pkgs.stdenv.isLinux then true else false;
```

Comment:
https://unix.stackexchange.com/a/390631

```
> bat ~/.config/systemd/user/ssh-agent.service
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: /home/kachick/.config/systemd/user/ssh-agent.service
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ [Install]
   2   │ WantedBy=default.target
   3   │
   4   │ [Service]
   5   │ ExecStart=/nix/store/11p34v2dfzjavry4kmwlp4d8jhjz4c6c-openssh-9.3p2/bin/ssh-agent -D -a %t/ssh-agent
   6   │
   7   │ [Unit]
   8   │ Description=SSH authentication agent
   9   │ Documentation=man:ssh-agent(1)
```

Already exists 🤔

Reply:
https://learn.microsoft.com/ja-jp/windows/wsl/systemd
https://forest.watch.impress.co.jp/docs/news/1441775.html

WSL problems 😂

Reply:
After it

```
> ps -ef | grep ssh
kachick      516     504  0 15:06 ?        00:00:00 /nix/store/11p34v2dfzjavry4kmwlp4d8jhjz4c6c-openssh-9.3p2/bin/ssh-agent -D -a /run/user/1000/ssh-agent
kachick     1225     652  0 15:07 pts/0    00:00:00 grep ssh
```